src/libraries/Fuzzing/README.md

```diff
@@ -18,28 +18,34 @@ Useful links:
 
 ### Prerequisites
 
-Build the runtime with the following arguments:
+Build the runtime if haven't already:
 ```cmd
-./build.cmd clr+libs+packs+host -rc Checked -c Debug
+./build.cmd clr+libs -rc release
```

Comment:
Why this change?

Using checked+debug was intentional so that we run with all debug asserts enabled, including in corelib

Reply:
Updated it mainly because before `+packs+host` was required to be built, but not now, just tried to show now it can run with a normal build, though your point to use checked for runtime by default makes sense, I can change that

Reply:
Not having to build `packs+host` will be nice to reduce build times, but I would keep it on checked+debug.

Reply:
though I suppose it doesn't really matter as much anymore, the build works no matter which one you build locally.